adsingestp/parsers/datacite.py

```diff
@@ -96,15 +96,18 @@ def _parse_contrib(self, author=True):
                     or i.get("schemeURI", "") == "http://orcid.org"
                 ):
                     for ct in contrib_tmp:
-                        ct["orcid"] = i.get_text()
+                        try:
+                            ct["orcid"] = i.get_text().replace("https://orcid.org/","")
+                        except Exception as err:
+                            ct["orcid"] = None
 
             if not author:
                 for ct in contrib_tmp:
                     ct["role"] = c.get("contributorType", "")
 
             contribs_out += contrib_tmp
-        if not contribs_out:
-            raise MissingAuthorsException("No contributors found for")
+        # if not contribs_out:
+        #     raise MissingAuthorsException("No contributors found for")
```

Comment:
Let's delete these lines instead of commenting them out, if they're not needed. The history will be here in Github anyway.